resolwe_bio/processes/clustering/hierarchical_clustering_etc.py

```diff
@@ -160,6 +160,7 @@ class Input:
         distance = StringField(
             label="Distance metric",
             choices=[
+                ("euclidean", "Euclidean"),
```

Comment:
`get_distance_metric` function should also handle this distance.

Ideally there would also be a small test for this.

Reply:
Input field `metric` of `scipy.cluster.hierarchy.linkage` takes the argument `'euclidean'`.